host/tests/integration_tests/python/runtime/http.rs

```diff
@@ -18,6 +18,46 @@ use crate::integration_tests::{
     test_utils::ColumnarValueExt,
 };
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_requests_simple() {
+    const CODE: &str = r#"
+import requests
+
+def perform_request(url: str) -> str:
```

Comment:
this shows the simplest possible HTTP usage now, which I think is kinda nice and elegant